<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SClncRNAFun</class>
 <widget class="QDialog" name="SClncRNAFun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1500</width>
       <height>800</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1500</width>
        <height>879</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>1291</width>
         <height>781</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1400</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="text">
           <string>NovelScan: Identify of novel transcripts (including mRNA and lncRNAs) and their characters </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1300</width>
            <height>230</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/images/novelscan.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>51</x>
             <y>418</y>
             <width>1181</width>
             <height>32</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="config">
              <property name="font">
               <font>
                <family>Arial [TMC ]</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Example Result</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="start">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Arial [TMC ]</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>START</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="config_2">
              <property name="font">
               <font>
                <family>Arial [TMC ]</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Result</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="next">
              <property name="font">
               <font>
                <family>Arial [TMC ]</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>NEXT</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>42</y>
             <width>1181</width>
             <height>371</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>Arial [TMC ]</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string> *Genome  --  (Select the species of the data )</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_genome">
                  <property name="font">
                   <font>
                    <family>Arial [TMC ]</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <property name="currentText">
                   <string>Human</string>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>Human</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mouse</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Fly</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Zebrafish</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Arial [TMC ]</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>output</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="line_out">
                  <property name="font">
                   <font>
                    <family>Arial [TMC ]</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="browse_data">
                  <property name="font">
                   <font>
                    <family>Arial [TMC ]</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Browse...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLineEdit" name="line_fa">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/hg38.fa</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_gtf">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/gencode.annotation.gtf</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_ERCC">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>ERCC92.gtf</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="line_protein">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/GENCODE_protein_coding_gene.gtf</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_lncRNA">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/GENCODE_long_noncoding_RNAs.gtf</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_model">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/human_model.hdf5</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLineEdit" name="line_pseudo">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/GENCODE_pseudogene.gtf</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_other">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/GENCODE_other_rnas.gtf</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="line_phastcons">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>hg38/hg38.phastCons.bw</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string> Coding Capability filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_cutoff">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>0.44</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>0.364</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string> TPM filter  --  (A transcript has at least three cells with a TPM greater than 0)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_fpkm">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string> Length filter  --  (Filter out transcripts whose length is less than the threshold)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_length">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>200</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QCheckBox" name="checkBox_exon">
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string> Exon Number filter  --  (By default, transcripts with exon number equal to 1 are removed)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_exon">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Arial [TMC ]</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
